superset-frontend/src/hooks/apiResources/queryApi.test.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import fetchMock from 'fetch-mock';
+import configureStore, { MockStore } from 'redux-mock-store';
+import rison from 'rison';
+import { JsonResponse } from '@superset-ui/core';
+import { supersetClientQuery } from './queryApi';
+
+const getBaseQueryApiMock = (store: MockStore) => ({
+  ...new AbortController(),
+  dispatch: store.dispatch,
+  getState: store.getState,
+  extra: undefined,
+  endpoint: 'endpoint',
+  type: 'query' as const,
+});
+
+describe('queryApi', () => {
```

Comment:
Could you avoid using `describe`? Check this [guideline](https://github.com/apache/superset/wiki/Testing-Guidelines-and-Best-Practices#avoid-nesting-when-youre-testing)